domain-models-runtime/src/main/java/org/folio/rest/persist/PostgresClient.java

```diff
@@ -2429,7 +2449,34 @@ private FacetManager buildFacetManager(CQLWrapper wrapper, QueryHelper queryHelp
    * @param replyHandler  the result; the JSON is encoded as a String
    */
   public void getByIdAsString(String table, String id, Handler<AsyncResult<String>> replyHandler) {
-    getById(table, id, string -> string, replyHandler);
+    getByIdAsString(null, table, id, replyHandler);
+  }
+
+  /**
+   * Get the jsonb by id and return it as a String. Query will be executed on
+   * <code>conn</code> if provided.
```

Comment:
The sentence `Query will be executed on <code>conn</code> if provided.` duplicates the information provided in the `@paran` entry and should be removed. The javadoc should be concise.